slave-base-ubuntu/Dockerfile

```diff
@@ -27,6 +27,12 @@ RUN apt-get update && \
     chmod -R 777 /opt/feedhenry/fh-dynoman/state/ && \
     mkdir -p /home/jenkins
 
+
+RUN apt-get -y install libcurl4-gnutls-dev libexpat1-dev gettext libz-dev libssl-dev build-essential make
```

Comment:
* Should L15 be updated to not install the git deb any longer?
* Would it be useful to add these 4 new `RUN` instructions to the one on L10-L28, rather than creating 4 new layers? I think it would; then after `make install`, you can remove the tarball and the extracted directory to keep the size down (it probably wouldn't be that much bigger with them, and probably isn't a big deal -- just highlighting that with individual `RUN` layers, even if you rm'ed those things in another `RUN`, it wouldn't make any difference to the size, unless you squashed the layers).